package.json

```diff
@@ -65,7 +65,7 @@
     "@otplib/core": "^12.0.1",
     "@otplib/plugin-base32-enc-dec": "^12.0.1",
     "aws-sdk": "^2.1356.0",
-    "axios": "^0.25.0",
+    "axios": "^0.28.0",
```

Comment:
Is it possible to upgrade this to the latest `axios` version, currently `1.6.8`?

Reply:
I've updated the version to 1.6.8 now, looks good asides from a change required in the https.ts file where it was enforcing to implement the full AxiosHeader properties